.gitignore

```diff
@@ -1,2 +1,2 @@
 node_modules/
-distribution/
```

Comment:
Maybe keep this in, because otherwise one might accidentally upload the old distribution folder from the local working directory.

Reply:
Yeah, I totally see that happening 👍